library/src/main/java/com/novoda/downloadmanager/VersionOneToVersionTwoMigrationStatus.java

```diff
@@ -0,0 +1,15 @@
+package com.novoda.downloadmanager;
+
+class VersionOneToVersionTwoMigrationStatus implements MigrationStatus {
```

Comment:
Just a simple wrapper for now, this will be built upon in a future PR when we decide what we want to surface to the user for the migration notifications.